src/helpers/percy.js

```diff
@@ -0,0 +1,47 @@
+const request = require('phin-retry');
+
+/**
+ * @param {import('../index').PercyAnalysisInputs} inputs 
+ */
+async function getProjectByName(inputs) {
+  return request.get({
+    url: `${inputs.url}/api/v1/projects`,
+    headers: {
+      'Authorization': `Token ${inputs.token}`
```

Comment:
maybe we should look at reading the Auth token from env variable later

Reply:
They can mention it using the special env syntax supported by this tool.

```js
"token": "{TOKEN}"
```